CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ SET(BUILD_SERVER ON CACHE BOOL
     "Build the Srch2Server executable. Defaults to ON")
 SET(BUILD_RELEASE ON CACHE BOOL "Build Release. Default ON")
 SET(BUILD_TEST ON CACHE BOOL "Build tests. Default ON")
-SET(BUILD_UTILITY OFF CACHE BOOL "Build utilities. Default OFF")
+SET(BUILD_UTILITY ON CACHE BOOL "Build utilities. Default ON")
```

Comment:
@surendra-bisht : please check if this change is related to the discussion we had with Saurav.